app/react-native/src/preview/components/OnDeviceUI/navigation/index.tsx

```diff
@@ -20,13 +19,19 @@ const style = StyleSheet.create({
   },
 });
 
-export default class Navigation extends PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      isUIVisible: props.initialUiVisible,
-    };
-  }
+interface Props {
+  initialUiVisible: boolean;
```

Comment:
According to https://github.com/storybooks/storybook/pull/6448/commits/ea0c8a0f4a997791ed4999c0f8d8b5cec487089a#diff-46b88ac54083f1f8e54a54e6f4a3088fL77
`initialUiVisible` is optional so maybe `initialUiVisible?: boolean;` is more accurate

Reply:
good catch